stdlib/crypto/src/main/ballerina/crypto/natives.bal

```diff
@@ -14,45 +14,119 @@
 // specific language governing permissions and limitations
 // under the License.
 
-# The hashing algorithms supported by this module.
-public type Algorithm SHA1|SHA256|MD5;
+# Returns the MD5 hash of the given data.
```

Comment:
Shall we rename this file to crypto.bal? AFAIR we discontinued the usage of natives. bal

Reply:
+1. natives.bal looked like the norm across stdlib